recipes/aaplus/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import functools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class Aaplusconan(ConanFile):
+    name = "aaplus"
+    description = (
+        "AA+ is a C++ implementation for the algorithms as presented in the "
+        "book \"Astronomical Algorithms\" by Jean Meeus"
+    )
+    license = "?"
```

Comment:
```suggestion
    license = "Unlicense"
```

https://spdx.org/licenses/Unlicense.html
The copyright listed on headers is pretty close to unlicense

Reply:
correct, forgot to edit this field :)